shared/engine/index.js

```diff
@@ -143,10 +143,10 @@ class Engine {
 
   // Default handlers for incoming messages
   _setupCoreHandlers() {
-    this.setIncomingHandler('keybase.1.logUi.log', (param, response) => {
+    this.setIncomingActionCreators('keybase.1.logUi.log', (param, response) => {
       const logParam: LogUiLogRpcParam = param
       log(logParam)
-      response && response.result && response.result()
+      if (response && response.result) return [response.result()]
```

Comment:
this isn't really an action, its just a callback thats passed on so it should likely be left alone in it's original form

Reply:
Meaning `setIncomingHandler` should be kept for this one use?

Reply:
no sorry, keep this line
```response && response.result && response.result()```

Reply:
Ah, okay. 👍